experimental/addons/gateway-traits/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+name: gateway-traits
+version: v1.0.0
```

Comment:
We should install the CRD about the Gateway API by this addon. Change the first version to v0.0.1.